content/registries/_index.md

```diff
@@ -13,3 +13,49 @@ weight: 23
 oc patch image.config.openshift.io/cluster -p \
 '{"spec":{"allowedRegistriesForImport":[{"domainName":"my.own.registry.example.com:8888","insecure":true}],"registrySources":{"insecureRegistries":["my.own.registry.example.com:8888"]}}}' --type='merge'
 ```
+
+# Configure custom/insecure registry to search path
+
+There seems to be no way to remove/modify or append any additional entries to unqualified-search-registries line in /etc/containers/registries.conf file as of now (OCP 4.3).
```

Comment:
Do you have the bugzilla/github issue where this will be fixed? If so, please add it as a reference

Reply:
At this moment there is no BZ or fix for OCP <= 4.3 (I think is in the roadmap for 4.4-5)

Reply:
:+1: